samples/nrf9160/modem_shell/src/rest/rest_shell.c

```diff
@@ -265,17 +276,31 @@ int rest_shell(const struct shell *shell, size_t argc, char **argv)
 	if (ret) {
 		mosh_error("Error %d from rest_lib", ret);
 	} else {
+		int headers_len = resp_ctx.response - req_ctx.resp_buff;
+		char *headers_start = '\0';
```

Comment:
Does this result in a pointer to a null or actually a null pointer? I think this could always point to `eq_ctx.resp_buff` and the printing could be just controlled using `printed_headers_len`. The assignment could then be removed from below.
```suggestion
		char *headers_start = req_ctx.resp_buff;
```

Reply:
Changed.